src/tclbinding.cpp

```diff
@@ -96,6 +90,7 @@ Napi::Value TclBinding::cmdSync( const Napi::CallbackInfo& info ) {
 	}
 
 	std::string cmd = info[0].As<Napi::String>().Utf8Value();
+//printf("I1:%p c:%s\n", this->_interp, cmd.c_str());
```

Comment:
Assume this was left over from debugging?

Reply:
Yes, sometimes I leave commented debug commands at key points where it has been handy to examine intermediate results.  I can take it out if you like.